test/fail_compilation/fail19099.d

```diff
@@ -0,0 +1,27 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19099.d(26): Error: cannot modify struct `a` `A` with `immutable` members
+---
+*/
+
+struct B
+{
+    this(this) {}
+    ~this() {}
+    int a;
+}
+
+struct A
+{
+    B b;
+    immutable int a;
```

Comment:
Does this error occur because `a` is immutable even though `b` isn’t?

Reply:
yes